tests/gds_kernel_latency.c

```diff
@@ -1540,11 +1538,11 @@ int main(int argc, char *argv[])
         if (!ctx)
                 return 1;
 
-        int nrecv = pp_post_recv(ctx, max_batch_len);
-        if (nrecv < max_batch_len) {
-                gpu_warn("[%d] Could not post all receive, requested %d, actually posted %d\n", my_rank, max_batch_len, nrecv);
-                return 1;
-        }
+//        int nrecv = pp_post_recv(ctx, max_batch_len);
```

Comment:
where is pp_post_recv() being called now?

Reply:
Inside `pp_post_work` before starting the main loop `for (i = 0; i < posted_recv; ++i)`. My initial question was: why there is this additional `pp_post_recv` outside and before the `pp_post_work` ?